src/navigators/WarningsTabIcon.tsx

```diff
@@ -2,12 +2,15 @@ import React from 'react';
 import Icon from '@components/common/Icon';
 
 import { selectCurrentDayWarningData } from '@store/warnings/selectors';
+import { selectTheme } from '@store/settings/selectors';
```

Comment:
Instead of importing theme from store, this component could use `useTheme` hook from `@react-navigation/native` which provides `dark` boolean.